flink-table/flink-table-planner/pom.xml

```diff
@@ -57,36 +57,12 @@ under the License.
 
 		<!-- core dependencies -->
 
-		<dependency>
-			<groupId>org.apache.flink</groupId>
```

Comment:
a pom should declare all dependencies that are directly used, so both `flink-table-common` and `flink-table-api-java`, and `bridge-base` are valid